README.md

```diff
@@ -32,6 +34,15 @@ FLASK_APP=app/__init__.py FLASK_DEBUG=1 flask run
 
 Open `http://localhost:5000` and log in as `user1`, `user2`, or `user3` with password `password`. Or login with `admin/admin`.
 
+***Frontend***
+```
+cd client 
+npm install
+npm run serve
+```
+
+To build the frontend into the flask server: ```./build-client.sh```
```

Comment:
maybe point out that the script is back at the top level of the repo since the reader might have just done `cd client`.

Reply:
Good point, I just updated it.